library/lib/stylesheet.js

```diff
@@ -0,0 +1,20 @@
+/*
+  Links to the stylesheet and adds hot reloading for stylesheets
+*/
+
+import hotCssReplacementClient from 'raw-loader!./hot-css-replacement-client'
+
+const isProduction = process.env.NODE_ENV === 'production'
+
+export default [
+  <link href={`/${__webpack_css_chunk_hash__}.css`} rel='stylesheet' type='text/css' key='hotCssReplacementClient_0' />,
```

Comment:
Perhaps we can add some `data-` attribute to this link-tag, in order to fix previous comment (https://github.com/kaliberjs/build/pull/63/files#diff-13df06f53a045c38c6befa6da7770b09R23)

Reply:
That's a really great idea! We could even adjust the selector to grab the correct link element directly (we know what hash it will be). Could you tell me what selector to use?

Reply:
Do you also know what hash it *was*? In that case something like 
```js
document.querySelector(`link[href="${previousHash}.css"]`).setAttribute('href', newHash)
```

Reply:
Might be necessary to have a special selector:
```css
/* Internal links, beginning with "#" */
a[href^="#"] {
  background-color: gold;
}

/* Links with "example" anywhere in the URL */
a[href*="example"] {
  background-color: silver;
}

/* Links with "insensitive" anywhere in the URL,
   regardless of capitalization */
a[href*="insensitive" i] {
  color: cyan;
}

/* Links that end in ".org" */
a[href$=".org"] {
  color: red;
}
```
https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors